lib/wovnrb.rb

```diff
@@ -109,7 +109,7 @@ def needs_api?(html_body, headers)
     end
 
     def wovn_ignored?(html_body)
-      !html_body.xpath('//html[@wovn-ignore]').empty?
+      !html_body.xpath('//html[@wovn-ignore]').empty? && !html_body.xpath('//html[@data-wovn-ignore]').empty?
```

Comment:
You can use `@wovn-ignore or @data-wovn-ignore`.

Reply:
Thank you! I did not know about that syntax until today :)

Reply:
To be honest, I also learned last week:stuck_out_tongue_closed_eyes: